packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -7,31 +7,49 @@ import Status from '@/components/Status';
 import { connectorTitlePlaceHolder } from '@/consts/connectors';
 
 import ConnectorName from '../ConnectorName';
+import * as styles from './index.module.scss';
 
 type Props = {
   type: ConnectorType;
   connector?: ConnectorDTO;
+  connectors?: ConnectorDTO[];
```

Comment:
I thought defining a `ConnectorDisplay` (maybe) type to handle this logic would be better.
Then we can map `ConnectorDTO` to `ConnectorDisplay` and use it here.

Reply:
We already have several types for connector, and add another type is not a good idea, it may be confusing.